src/views/PublicationRequest.js

```diff
@@ -52,7 +55,10 @@ const PublicationRequest = ({ handlers, data: { publicationRequest: request } =
       <RequestInfo request={request} />
```

Comment:
I might exchange the repeated `request={request}` props with a single method:
```getSectionProps = () => ({
 request
})
```
And then spread that onto each of them. That way you have a single centralised place to hand repeated props to all children. Similar to how we do it in agreements